### PR TITLE
Get available translations

### PR DESCRIPTION
**Pull request by Qubad786**

### CODE REVIEW

**Comment by Qubad786** [comment]
@muhammad-ammar have a look.

File: edxval/api.py [hunk 1]
```diff
@@ -222,6 +222,23 @@ def get_video_transcript_data(video_ids, language_code):
...
+    Arguments:
+        video_ids(list): list containing edx_video_id and external video ids extracted from
+        external sources of a video component.
+
+    Returns:
+        A list containing unique transcript language codes for the video ids.
+    """
+    available_languages = (VideoTranscript.objects
+                           .filter(video_id__in=video_ids)
+                           .values_list('language_code', flat=True))
```
Comment (muhammad-ammar) [inline edxval/api.py]:
```python
available_languages = VideoTranscript.objects.filter(
    video_id__in=video_ids
).values_list(
    'language_code',
    flat=True
)
```

**Comment by muhammad-ammar** [comment]
@Qubad786 Are you writing tests for this?

**Comment by Qubad786** [comment]
@muhammad-ammar yes.

**Comment by Qubad786** [comment]
@muzaffaryousaf @muhammad-ammar this is ready.

**Pull request merged**